rust/main/agents/scraper/migration/src/m20230309_000001_create_table_domain.rs

```diff
@@ -490,7 +490,7 @@ impl MigrationTrait for Migration {
                     .col(ColumnDef::new(Domain::TimeUpdated).timestamp().not_null())
                     .col(ColumnDef::new(Domain::Name).text().not_null())
                     .col(ColumnDef::new(Domain::NativeToken).text().not_null())
-                    .col(ColumnDef::new(Domain::ChainId).big_unsigned().unique_key())
+                    .col(ColumnDef::new(Domain::ChainId).big_unsigned())
```

Comment:
any idea if it's easy to re-generate this code? iiuc it's generated from these migrations, not sure the implications of not updating it https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/main/agents/scraper/src/db/generated/domain.rs#L68

Reply:
gotcha, forgot about this, makes sense!